ocs_ci/templates/ocs-deployment/storage-cluster.yaml

```diff
@@ -5,6 +5,15 @@ metadata:
   namespace: openshift-storage
   name: ocs-storagecluster
 spec:
+  managedResources:
```

Comment:
This is just from 4.8 and this file is used also in other version.

Please check if we deploy via UI if those are even defined, I think we are trying to keep this file as minimal as possible and this should use the default from rook operator if not defined in storage cluster CR